tests/test_data_languages.py

```diff
@@ -265,3 +265,10 @@ def test_exemplar_parser():
         "l",
         "̍",
     }
+
+
+def test_language_uniqueness():
+    names = Counter([lang.name for lang in LANGUAGES.values()])
```

Comment:
When present, I think you want to use `lang.preferredName` instead of `lang.name`.

That said, you might consider using that `preferredName` field in some of the language proto files in this PR. I don't actually see specific examples where that would make more sense, but worth considering.

Reply:
Oh boy, that found extra duplicates. :-)